flokk_src/lib/_internal/components/animated_panel.dart

```diff
@@ -8,9 +7,16 @@ class AnimatedPanel extends StatefulWidget {
   final double closedY;
   final double duration;
   final Curve curve;
-  final Widget child;
+  final Widget? child;
```

Comment:
This should be required, since a key part of this widget is that it handles hiding for you, you can always  just pass the child.